src/content/guides/hot-module-replacement.md

```diff
@@ -45,9 +46,8 @@ __webpack.config.js__
 
   module.exports = {
     entry: {
--      app: './src/index.js',
+       app: './src/index.js'
```

Comment:
now compare this line: you've changed it, but it's not reflected in the diff
does this PR make sense?

Reply:
The following sentence is written in the markdown file : "We'll also remove the entry point for print.js..."

So I guess this should be the correct example 
<img width="677" alt="screen shot 2017-12-15 at 22 36 48" src="https://user-images.githubusercontent.com/12511178/34061436-9e17ca12-e1e8-11e7-8d5a-f2a8e019a3a7.png">


And this is the current example
<img width="874" alt="screen shot 2017-12-15 at 22 38 29" src="https://user-images.githubusercontent.com/12511178/34061464-c56562dc-e1e8-11e7-9ce8-33bd66904350.png">

Which removes the `app.js` line and then re-adds it.. Which is a step to much I guess